buildpipeline/pipeline.json

```diff
@@ -58,20 +58,8 @@
         {
           "Name": "DotNet-CoreFx-Trusted-Linux",
           "Parameters": {
-            "PB_DockerTag": "opensuse132_prereqs_v4",
-            "PB_TargetQueue": "Suse.132.Amd64"
-          },
-          "ReportingParameters": {
-            "OperatingSystem": "openSUSE 13.2",
-            "Type": "build/product/",
-            "ConfigurationGroup": "Release"
-          }
-        },
-        {
-          "Name": "DotNet-CoreFx-Trusted-Linux",
-          "Parameters": {
             "PB_DockerTag": "opensuse421_prereqs_v3",
-            "PB_CreateHelixArguments": "/t:CloudBuild /p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:\"EnableCloudTest=false\" /p:\"TestProduct=corefx /p:\"TimeoutInSeconds=1200\" /p:\"TargetOS=Linux\" /p:FilterToOSGroup=Linux /p:\"BuildMoniker=none\""
```

Comment:
The line removal called my attention to the fact it is using some arguments that are not interesting any more. In particular FilterToOSGroup and TargetOS. While you removed this instance we have a number of instances in the file that appear to use them which should get cleaned up. Feel free to do that as another PR.